### PR TITLE
milestone 1 initial commit

### PR DESCRIPTION
**Pull request by grace-mugoiri**

### CODE REVIEW

File: bookstore/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	  ],
+	  "space-before-function-paren": [
+		"error",
+		{ "anonymous": "always", "named": "never", "asyncArrow": "always" }
+	  ],
+	  "indent": ["error", 2],
+	  "no-trailing-spaces": ["error", { "skipBlankLines": true }],
+	  "no-console": ["error", { "allow": ["warn"] }]
+	}
+  }
```
Comment (jofer86) [inline bookstore/.eslintrc.json]: - This .eslintrc.json file is not the one provided by Microverse, please refer to [the Microverse linter repo for react/redux](https://github.com/microverseinc/linters-config/tree/master/react-redux); for a proper configuration of our linter configuration files. As the file should look like this: 
image

File: bookstore/eslint.config [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+		"import/extensions": "off",
+		"arrow-parens": [2, "as-needed"],
+		"import/no-named-as-default": 0,
+		"no-named-as-default-member": 0,
+		"no-nested-ternary": 0,
+		"no-unused-expressions": 0,
+		"no-restricted-globals": 0,
+		"import/no-unresolved": "off"
+    }
+}
```
Comment (jofer86) [inline bookstore/eslint.config]: Please do delete this file as it is not going to be used.

File: bookstore/.stickler.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+linters:
+  flake8:
+  eslint:
+    config: './eslint.config'
+  stylelint:
+files:
+  ignore:
+    - '.eslintrc.json'
```
Comment (jofer86) [inline bookstore/.stickler.yml]: This file should look like the one in the [Microverse Linters Repo](https://github.com/microverseinc/linters-config/blob/master/react-redux/.stickler.yml)
image

**Review by jofer86 — CHANGES_REQUESTED** [review]
## :wave: :wave: Hello, I'm here to the review on this PR.
- Below you are going to find some changes that are **required** for you to do in order for this project to be approved, please tend to them.
- You need to add a descriptive README file to your project as that is also a Hard Requirement to consider this project a valid one. You can read all about it [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
--------------------------------------------------------------------
Cheers, 
Jorge Fernando Rincón Figueroa.
Technical Support Engineer.
**Slack: @jofer86**

File: src/serviceWorker.js [hunk 4]
```diff
@@ -11,13 +11,13 @@
 // opt-in, read https://bit.ly/CRA-PWA
 
 const isLocalhost = Boolean(
-  window.location.hostname === 'localhost' ||
+  window.location.hostname === 'localhost'
```
Comment (bolah2009) [inline src/serviceWorker.js]: [Optional]

I would recommend deleting this file

**Review by bolah2009 — APPROVED** [review]
Hi @grace-mugoiri 

@jofer86  seems to miss this but you need a descriptive PR description. Please see [this pathwright lesson for detailed instructions](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)

> Pull Request has meaningful title and a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

It's a 👍🏾 from me!

### :heavy_check_mark: Approved!

Good luck with your next milestones. :star2: 

Please see the comment below and update PR description before merging.

Cheers
[Bola](https://bolabuari.com/)

**Pull request merged**